gulpfile.js

```diff
@@ -103,9 +103,9 @@ const hygiene = exports.hygiene = (some, options) => {
             .split(/\r\n|\r|\n/)
             .forEach((line, i) => {
                 if (/^\s*$/.test(line)) {
-                    // empty or whitespace lines are OK
+                    // empty or whitespace lines are OK.
```

Comment:
If you want to make the comments be properly formatted, capitalization is probably worth it (for the same reasons for adding a period; you know no one accidentally deleted a comment line).

Reply:
Fixed